static/defaults.config.php

```diff
@@ -53,15 +53,10 @@
 		// Database connection charset. Changing this value will likely corrupt special characters.
 		'charset' => 'utf8mb4',
 
-		// emulate_prepares (Boolean) (Experimental)
-		// If enabled, prepared statements will be emulated.
-		// In combination with MySQLi this will cast all return values to strings.
-		'emulate_prepares' => false,
-
-		// pdo_emulate_prepares (Boolean) (Experimental)
+		// pdo_emulate_prepares (Boolean)
 		// If enabled, the builtin emulation for prepared statements is used.
-		// Due to limitations of that emulation (all return values are casted as strings)
-		// this will most likely cause issues and should not be used on production systems.
+		// When enabled, a workaround is used to ensure that the returned field types are correct,
```

Comment:
Please replace these technical considerations that aren’t usable for admins (since we made the setting transparent from a variable type standpoint) and explain the case where its value should be changed from the default with the relevant environment information and the error it solves.

Reply:
Done